framework/src/middlewares/success-check.ts

```diff
@@ -0,0 +1,54 @@
+import staticImplements from '../helper/static-implements.js';
+import Middleware from '../middleware.js';
+import Result from '../result.js';
+import language from '../helper/language.js';
+import StandardResponse from '../standard-response.js';
+
+/*
+Below Document Defines the standards upon which most apis will be constructed
+for success scenarios :
+
+1) JSEND - https://github.com/omniti-labs/jsend
+{
+  "status": "success",
+  "data":{},
+}
+
+2) Another Common Format
+{
+  "success": true,
+  "data": {}
+}
+*/
+
+const FAIL = 'fail';
+const ERROR = 'error';
```

Comment:
please just integrate the string, single use constants for strings have little benefit.